layouts/articles/single.html

```diff
@@ -1,5 +1,5 @@
-{{ define "main" }} {{ $author_page := (printf "/authors/%s/index.md"
-.Params.author) }}
+{{ define "main" }} 
+{{ $author_page := (printf "/authors/%s/index.md".Params.author) }}
```

Comment:
I ended up creating another PR to fix this line as well. I accidentally didn't add a space before `.Params.author`.